### PR TITLE
trigger re-connect after RPC error.

### LINKED ISSUE #69: Client fails to reconnect
If the client library's connection to the frontend breaks, it keeps trying to use the broken connection.

```
[FrontendRPC:RPC] frontend_rpc.go:49: rpc error: unexpected EOF
[FrontendRPC:RPC] frontend_rpc.go:49: rpc error: unexpected EOF
[FrontendRPC:RPC] frontend_rpc.go:49: rpc error: connection is shut down
[FrontendRPC:RPC] frontend_rpc.go:49: rpc error: connection is shut down
...
```

### PR DESCRIPTION
**Pull request by willscott**
fix #69

### CODE REVIEW

File: common/frontend_rpc.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ func (f *FrontendRPC) Call(methodName string, args interface{}, reply interface{
 	err = f.client.Call(methodName, args, reply)
 	if err != nil {
 		f.log.Printf("rpc error: %v", err)
+		f.client = nil
```
Comment (ryscheng) [inline common/frontend_rpc.go]: Do we need to call Close()?

**Comment by ryscheng** [comment]
Also as an FYI, I've abstracted out the calling code in my branch, for future rpc stubs to use.
https://github.com/privacylab/talek/blob/ryscheng-coordinator/common/rpc_call.go

I'll update my branch to do the same

File: common/frontend_rpc.go (hunk 1 above)
Comment (willscott) [inline common/frontend_rpc.go]: can't hurt. done.

**Comment by willscott** [comment]
awesome.

Also: the RPC wire protocol is weirdly verbose. should we think about protobufs or something?

**Pull request merged**